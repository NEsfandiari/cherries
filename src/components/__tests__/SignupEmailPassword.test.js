import React from 'react'
import renderer from 'react-test-renderer'
import SignupEmailPassowrd from '../molecules/SignupEmailPassword'

describe('SignupEmailPasstowd', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<SignupEmailPassowrd />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
