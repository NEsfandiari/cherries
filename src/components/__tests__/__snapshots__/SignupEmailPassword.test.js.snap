// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignupEmailPasstowd renders correctly 1`] = `
<form
  className="sc-jTzLTM rVHQv"
  onSubmit={[Function]}
>
  <div
    className="name"
  >
    <input
      aria-label="First Name"
      className="sc-htoDjs kLyOFi"
      fontFamily="Lato"
      name="firstName"
      onChange={[Function]}
      placeholder="First Name"
      type="text"
      value=""
    />
    <input
      aria-label="Last Name"
      className="sc-htoDjs cBJBGT"
      fontFamily="Lato"
      name="lastName"
      onChange={[Function]}
      placeholder="Last Name"
      type="text"
      value=""
    />
  </div>
  <input
    aria-label="Email Address"
    className="sc-htoDjs iDUgja"
    fontFamily="Lato"
    name="email"
    onChange={[Function]}
    placeholder="Email"
    type="email"
    value=""
    width="24.3rem"
  />
  <input
    aria-label="Password"
    className="sc-htoDjs kDewdI"
    fontFamily="Lato"
    name="password"
    onChange={[Function]}
    placeholder="Password"
    type="password"
    value=""
    width="24.3rem"
  />
  <button
    className="sc-gzVnrw eVkDhi"
    fontSize="16px"
    fontWeight="bold"
    height="2.2rem"
    width="25.5rem"
  >
    SIGN UP
  </button>
  <div
    className="radio"
  >
    <input
      aria-label="Newsletter"
      checked={false}
      className="sc-htoDjs jwCMsF"
      id="newsletter"
      name="newsletter"
      onClick={[Function]}
      type="radio"
      width="2rem"
    />
    <label
      for="newsletter"
    >
       I'd like to hear from Cherries
    </label>
  </div>
</form>
`;
